src/libsyntax/parse/token.rs

```diff
@@ -284,7 +284,6 @@ impl Token {
         match self {
             OpenDelim(Brace) => true,
             Interpolated(ref nt) => match nt.0 {
-                NtExpr(..) => true,
```

Comment:
This was ok, interpolated expressions are "indivisible" / effectively parenthesized, so the angle bracket issues do not affect them.

Reply:
Ah, okay. I'll check that back.